packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.cnstyles.ts

```diff
@@ -225,6 +225,9 @@ export const getMenuItemStyles = memoizeFunction(
           [MediumScreenSelector]: {
             fontSize: IconFontSizes.medium, // 16px
           },
+          [HighContrastSelector]: {
+            color: 'HighlightText',
```

Comment:
This should be WindowText color

Reply:
@xugao  The background in high contrast is `Highlight`, so should the text color be `HighlightText`, similar to how it is for the root text and for `iconColor`? 

I tested locally setting it to `WindowText`, and in high contrast white, the icon looks similar to the "Before" picture in the description, where there's not much contrast with the background.

Reply:
Ah you are right.

Reply:
I merged the change for you:)